crates/sp-domains-fraud-proof/src/host_functions.rs

```diff
@@ -262,6 +262,8 @@ where
         domain_inherent_extrinsic_data: DomainInherentExtrinsicData,
     ) -> Option<DomainInherentExtrinsic> {
         let DomainInherentExtrinsicData {
+            // Used by caller
+            block_randomness: _,
```

Comment:
This PR contains a host function change, which we are planning to revert in #3314